src/accelerator/ogl/image/shader.frag

```diff
@@ -509,6 +510,23 @@ vec4 get_rgba_color()
         return vec4(get_sample(plane[0], TexCoord.st / TexCoord.q).bgr, 1.0);
     case 9:		//rgb,
         return vec4(get_sample(plane[0], TexCoord.st / TexCoord.q).rgb, 1.0);
+    case 10:	//uyvy,
+        {
+            float y = get_sample(plane[0], TexCoord.st).g;
+            float cb;
+            float cr;
+            if(mod(TexCoord.s*texture_width, 2.0) < 1.0)
```

Comment:
Is there any way this could be done without an if statement?
From what I remember if statements like this are real performance killers

Reply:
@Julusian  Maybe:
`float posx = round(mod(TexCoord.s * texture_width, 2.0));
cb = textureOffset(plane[0], TexCoord.st, ivec2(0 - posx, 0)).r;
cr = textureOffset(plane[0], TexCoord.st, ivec2(1 - posx, 0)).r;`

Reply:
#1288